content/blog/devtools_ui.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "Chrome DevTools UI"
+date: 2021-01-11T09:00:00+02:00
+description: "Chrome DevTools has always been an essential feature in Sauce Labs for live testing real devices. This blog post reports about our recent revamp of the UI code."
+author: "Sakhi Mansoor"
+images:
+    - /images/blog/devtools-ui/after.jpg
+tags:
+    - ux
+    - frontend
+    - chrome
+    - devtools
+    - devtools-ui
+---
+
+Chrome DevTools has always been an essential feature in Sauce Labs for live testing real devices. This will allow a user to inspect web pages in Android devices during live testing and use Chrome DevTools to debug issues.
+
+## Background
+
+In 2016, we launched a Chrome DevTools application to our customers that enabled them to debug webpages in Chrome and Safari in Android and iOS devices respectively. We used the static asset provided by the Chromium team and cloned the whole repository of [artifacts](https://www.google.com/url?q=https://github.com/ChromeDevTools/devtools-frontend/tree/master/front_end&sa=D&ust=1610370981658000&usg=AOvVaw0F69KLsKNRZ2R6ENTiGAKs) in our product from their git source to render the DevTools UI. Some other browsers which use the Chromium DevTools protocol were also integrated with some limitations.
```

Comment:
The `artifacts` redirects to this rather than directly  navigating to that git page.


![image](https://user-images.githubusercontent.com/48210502/104203196-27ebef80-542c-11eb-9c56-aa92d111300d.png)